src/ctapipe/image/muon/intensity_fitter.py

```diff
@@ -317,7 +344,12 @@ def image_prediction_no_units(
     # diameter. In any case, since in the end we do a data-MC comparison of the muon
     # ring analysis outputs, it is not critical that this value is exact.
```

Comment:
The comment here provided by @moralejo is now longer correct, since you introduce the correction for the pixel shape.

Please update it.

Please also have a look here @moralejo.